.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,58 @@
+name: Feature Request
+description: Suggest a new feature and/or improvement for Citizens.
+
+body:
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ## DO NOT IGNORE
+        Please fill out all required fields the way they are described as.
+        **Not following the issue form will get it closed automatically!**
+        
+        ----
+        
+        ### Section 1) Checks
+        Make sure you've done the following checks before submitting a feature request.
+  - type: 'checkboxes'
+    attributes:
+      label: 'I confirm that I have ...'
+      description: 'Click the checkboxes to "check" them. Make sure you actually did what they mention.'
+      options:
+        - label: '...tried the latest [Development Build](https://wiki.citizensnpcs.co/API) to make sure my feature doesn''t exist already.'
```

Comment:
I'm pretty sure the intended link was https://ci.citizensnpcs.co/job/Citizens2/ not the API wiki page?